packages/board/package.json

```diff
@@ -10,8 +10,8 @@
   },
   "devDependencies": {},
   "dependencies": {
-    "@botpress/webchat": "0.0.1",
-    "@botpress/webchat-skin": "0.0.1",
+    "@botpress/chat": "0.0.1",
+    "@botpress/chat-skin": "0.0.1",
```

Comment:
The only other change in this PR is renaming the packages for the previous test webchat to `chat` instead of `webchat` to avoid conflicting names.